src/libraries/Microsoft.Windows.Compatibility/src/Microsoft.Windows.Compatibility.csproj

```
@@ -5,8 +5,8 @@
     <!-- Reference the outputs for the dependency nodes calculation. -->
     <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
     <IsPackable>true</IsPackable>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-    <ServicingVersion>6</ServicingVersion>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>7</ServicingVersion>
```

Per discussion - STJ isn't actually referenced by this so it wasn't missed.
```suggestion
    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
    <ServicingVersion>6</ServicingVersion>
```